Traceback (most recent call last):
  File "../../Converter/main.py", line 166, in <module>
    main()
  File "../../Converter/main.py", line 150, in main
    dep_tree = sys.argv[1].encode("utf-8")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd7 in position 24: ordinal not in range(128)
